### DISCUSSION TITLE
macOS support

### DISCUSSION THREAD

**Discussion by fennabe** [m1dugh/native-sound-mixer#15] [Q&A]
Just wanted to know what your thoughts are on supporting macOS. I've done a lot of research about per-application volume control on macOS but it seems a really difficult task. It might require some sort of virtual device or special driver...

I noticed you just finished Linux support. What are your thoughts on macOS support? Do you already have a plan how you would do this? I'm happy to help in any way but I'm not a C++ or macOS developer.. (I do mostly javascript) I only did a lot of reading about this topic :)

macOS support would be awesome since there are no good solutions as far as I know.

**Comment by m1dugh** [comment]
I didn't get myself too much into the mac os part but i noticed pulseaudio worked on mac os too so if needed you could recompile the linux binaries for mac os with a pulseaudio server and it might work (i didn't check, can only guess). Happy you read about the subject, if you want you could write some kind of very abstract pseudo-code that would explain the path to take but what i'm lacking the most is a mac os iso to run a vm for the tests. Feel glad you took time for that project.
